openapi/listings/listings.yaml

```diff
@@ -447,3 +571,315 @@ components:
         The listing score information for a business location
       x-tags:
         - Listing Scores
+    listingProfiles:
+      type: object
+      x-examples:
+        Example 1:
+          id: AG-1234567
+          type: businessLocations
+          attributes:
+            customerIdentifier: string
+            name: string
+            address:
+              line1: 109 8th Street E.
+              line2: Suite 23
+              city: string
+              stateCode: CA
+              zip: string
+              postalCode: S7M 1R3
+              regionCode: CA-SK
+              countryCode: CA
+            phoneNumbers:
+              - string
+            serviceAreaBusiness: true
+            geoCoordinate:
+              latitude: -90
+              longitude: -180
+            hours:
+              - hoursTypeId: general
+                regularHours:
+                  - openDay: wednesday
+                    closeDay: wednesday
+                    openTime: '09:45'
+                    closeTime: '09:45'
+                specialHours:
+                  - status: closed
+                    startDate: '2019-08-24'
+                    endDate: '2019-08-24'
+                    openTime: '09:45'
+                    closeTime: '09:45'
+            updatedAt: string
```

Comment:
The `updatedAt` field is missing a description, readonly flag and type specification. Let's leave it out for now as I suspect that it changes every time the account group is updated instead of just when one of the fields on the listing profile is updated. 

```suggestion
```